compare-bug-version-and-git-branch.sh

```diff
@@ -20,7 +20,7 @@ if ! git show --pretty=format:%B | head -n 2 | tail -n 1 | egrep '^$' >/dev/null
 fi
 
 BUG=$(git show --name-only --format=email | awk '{IGNORECASE=1} /^BUG:/{print $2}' | tail -1)
-if [ -z "${BUG}" -a "${GERRIT_TOPIC}" = "rfc" ]; then
+if [ -z "${BUG}" -a "${GERRIT_TOPIC#*rfc*}" != "${GERRIT_TOPIC}" ]; then
```

Comment:
Considering we're targetting this for release branch, I'd say, let's add a second check after this, which checks that branch name starts with "release-" and has rfc at the end of the topic.